Tools/CISupport/ews-build/steps.py

```diff
@@ -3780,8 +3780,8 @@ def __init__(self, identifier=''):
         if identifier and not identifier.startswith('-'):
             identifier = '-{}'.format(identifier)
 
-        self.zipFile = Interpolate('public_html/results/%(prop:buildername)s/r%(prop:patch_id)s-%(prop:buildnumber)s{}.zip'.format(identifier))
-        self.resultDirectory = Interpolate('public_html/results/%(prop:buildername)s/r%(prop:patch_id)s-%(prop:buildnumber)s{}'.format(identifier))
```

Comment:
I wonder if removing this `r` will break anything for patch workflow. Can you please test that on uat instance.